cargo-dist/src/backend/installer/homebrew.rs

```diff
@@ -369,6 +369,9 @@ mod tests {
     fn spdx_three_licenses_or_and() {
         run_spdx_comparison(
             "MIT OR Apache-2.0 AND CC-BY-4.0",
+            // NOTE: Homebrew parses this as {:all_of=>[{:any_of=>["MIT", "Apache-2.0"]}, "CC-BY-4.0"]}
+            // According to the spec, this seems to be wrong -
+            // the result produced here is correct.
```

Comment:
Not sure what your preferences are around comment verbosity, but I've suggested an explanation and the spec URL in case you'd like to use any of it.

```suggestion
            // NOTE: Homebrew parses this as {:all_of=>[{:any_of=>["MIT", "Apache-2.0"]}, "CC-BY-4.0"]}
            // According to the SPDX v3 spec, this seems to be wrong, as operator precedence is specified
            // as WITH > AND > OR (while Homebrew evaluates OR operators, then AND operators).
            // The result produced in this test is correct.
            //
            // https://spdx.github.io/spdx-spec/v3.0/annexes/SPDX-license-expressions/#d45-order-of-precedence-and-parentheses
```

Reply:
Perfect, thanks!